### ISSUE TITLE
Squig Herder Summon Squig takes ya out of Squig Armor

### ISSUE THREAD

**Issue by Monzsta** [WarEmu/WarBugs#18439]
Hi Devs,

Found new Bug: When you are in Squig Armor and Summon any Squig Pet you get taken out of Squig Armor.

cheers Monzta

**Comment by Zydna** [comment]
Yep, but I think it is connected to the tactic Sneaky Git. It seems like when I use that and summon a pet in SA, it takes me out of it.
Also, Sneaky Git doesn't show correct increased defenses in statdoll.

**Comment by Monzsta** [comment]
i can confirm what you say, same here

**Comment by Zydna** [comment]
Sneaky Git tactic have been very buggy since it was added.

**Comment by Zydna** [comment]
Is this being looked at?

**Comment by barfyman92** [comment]
Wow this issue is 20 days old! I just tried to play my MSH it and it was annoying as hell.

**Comment by Zydna** [comment]
yea the tactic can't be used atm.

**Comment by jyknn** [comment]
Bump, this bug has been going on for too long now and really harms the gameplay of MSH. Also because of that you can't eat your pet during the fight, removing one of our skill. 
Very annoying bug please check out this sneaky git tactic

**Comment by Tohrgrimm** [comment]
Can confirm the here listed problems. In Summary: Sneaky Git tactic bugs you always out of the Squig armor, so you cant play melee squig the way you should. Sneaky Bouncin tactic does not work, no damage increase. At least 2 tactics are not working.

**Comment by Tohrgrimm** [comment]
Tested it today again and the here explained problem still exists

**Comment by Killham** [comment]
I can confirm that problem still here. It is unplayable if you melee sh with tactic "sneaky git" and want to play with pet, its dropping you from squig armor every time when you summoning pet with this tactic.

**Comment by SilverWF** [comment]
Still has this issue...
